iotedgedev/modules.py

```diff
@@ -36,10 +41,24 @@ def add(self, name, template):
         elif os.path.exists(os.path.join(cwd, name)):
             raise ValueError("Module \"{0}\" already exists under {1}".format(name, os.path.abspath(self.envvars.MODULES_PATH)))
 
-        deployment_manifest = DeploymentManifest(self.envvars, self.output, self.utility, self.envvars.DEPLOYMENT_CONFIG_TEMPLATE_FILE, True)
-
         repo = "{0}/{1}".format("${CONTAINER_REGISTRY_SERVER}", name.lower())
-        if template == "csharp":
+        if template == "c":
+            github_prefix = "https://github.com/Azure"
+            git_repo = "azure-iot-edge-c-module"
+            branch = "master"
+            url = "{0}/{1}/archive/{2}.zip".format(github_prefix, git_repo, branch)
```

Comment:
Are we doing this because Nuget was removed from the C SDK repo?

Reply:
No related to the distribution of SDK. For Node scaffolding, we use Yeoman. For python scaffolding, we use Cookiecutter. For C, there is no appropriate scaffolding tool, so we use github to host the C template fot IoT Edge Module.

Reply:
Sorry, my comments were due to me thinking this was the actual IoT C SDK.